### ISSUE TITLE
ACCESSIBILITY: readonly fields should be <input readonly >

### ISSUE THREAD

**Issue by tractorcow** [silverstripe/silverstripe-admin#14]
At the moment we use `<FormControl.Static />` in react which doesn't produce proper input tags. I suggest that we should probably stick with literal inputs for all input-type fields, and flag these with either `disabled` or `readonly` html properties.

This distinguishes readonly inputs from literals (provided by LiteralField).

**Comment by dhensby** [comment]
is this going to be actioned?

**Comment by tractorcow** [comment]
At some point yeah. When I get more time. :)

**Comment by GuySartorelli** [comment]
Closed in favour of https://github.com/silverstripe/silverstripe-admin/issues/1963
